t/plugin/traffic-split2.t

```diff
@@ -268,7 +268,7 @@ x-real-ip: 127.0.0.1
                                                 "type": "roundrobin",
                                                 "pass_host": "node",
                                                 "nodes": {
-                                                    "localhost:1981":1
```

Comment:
Why change it?

Reply:
Sorry, this is because before I submitted this PR, I modified the `localhsot` mapping address of the local `hosts` file. Therefore, the test case of this PR failed when running, so I modified the test case, but I forgot to restore it.

Reply:
Updated.